generate-votes-list/voting-version2.js

```diff
@@ -93,30 +114,30 @@ async function setupProgressBar() {
 }
 
 function checkConfig() {
-    if (fs.existsSync("./.cfg") && fs.existsSync("./votes.json")) {
+    if (fs.existsSync("./.cfg") && fs.existsSync("./ethVotes.json")) {
       let input = fs.readFileSync("./.cfg");
       let jsonConfig = JSON.parse(input);
       lastCount = jsonConfig.lastCount;
-      input = fs.readFileSync("./votes.json");
+      input = fs.readFileSync("./ethVotes.json");
       json = JSON.parse(input);
     }
 }
 
 function saveJSON() {
-    let jsonString = JSON.stringify(json, Object.keys(json).concat(["vote", "totalBalance", "burnedTokens", "ownedTokens"]), 2);
+    let jsonString = JSON.stringify(json, null, 2);
     jsonString = jsonString.replace(/: "/gm, ': ')
```

Comment:
these 3 lines should be removed, as it produces invalid json

Reply:
I'm getting fine results with the original, and screwed json with your changes @thepiwo